### PR TITLE
Fixes missing candles in the candle boxes around the station

### LINKED ISSUE #12294: Empty candle boxes around the chapel
**Issue Description**:
The candle boxes in the chapel and in chapel maint are empty but their icons show that they have candles.

**What did you expect to happen**: 
For there to be candles in the boxes or at least that the boxes look empty at round start.

**What happened instead**:
The candle boxes don't have candles in them but look like they have candles until you put an item in, then it updates the sprite.

**Why is this bad/What are the consequences**:
Chaplains sometimes like to use candles for the chapel and others may want to grab a box for personal use.

**Steps to reproduce the problem**:
Check all candle boxes around the chapel and you'll find them empty(the ones in the chapel and the ones immediately outside in chapel maint. The ones in the chaplain locker are fine). The candle boxes at the bar are fine however(meaning they have candles in them)

**When did the problem start happening**:
This I don't know but found out about it on September 13 2019

**Extra information**:
None that I can think of.

### PR DESCRIPTION
**Pull request by florkeger**
Fixes #12294

## What Does This PR Do
Look at the title.

## Why It's Good For The Game
It fixes the boxes, duh.

## Changelog
:cl: kazboo
fix: candle boxes on the station are now properly filled
/:cl:

### CODE REVIEW

**Comment by Mitchs98** [comment]
Let there be light.

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -31966,15 +31966,9 @@
...
-	pixel_x = -3;
-	pixel_y = 3
-	},
-/obj/item/storage/fancy/candle_box{
-	pixel_x = 3;
-	pixel_y = -3
-	},
+/obj/item/storage/fancy/candle_box/full,
+/obj/item/storage/fancy/candle_box/full,
+/obj/item/storage/fancy/candle_box/full,
```
Comment (Fox-McCloud) [inline _maps/map_files/cyberiad/cyberiad.dmm]: You wiped out their offsets. Why?

Comment (florkeger) [reply _maps/map_files/cyberiad/cyberiad.dmm]: When I checked them on the test server, they still had their off-sets. Don't know why it did that.

Comment (Fox-McCloud) [reply _maps/map_files/cyberiad/cyberiad.dmm]: Fix this, please.

Comment (florkeger) [reply _maps/map_files/cyberiad/cyberiad.dmm]: Is fixed.

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 2]
```diff
@@ -31966,13 +31966,13 @@
...
-/obj/item/storage/fancy/candle_box{
+/obj/item/storage/fancy/candle_box/full{
 	pixel_x = -3;
 	pixel_y = 3
 	},
-/obj/item/storage/fancy/candle_box{
-	pixel_x = 3;
+/obj/item/storage/fancy/candle_box/full,
+/obj/item/storage/fancy/candle_box/full{
+	pixel_x = -3;
```
Comment (variableundefined) [inline _maps/map_files/cyberiad/cyberiad.dmm]: This one should be a 3 I believe

Comment (florkeger) [reply _maps/map_files/cyberiad/cyberiad.dmm]: I made it a -3 so they aren't stacked awkwardly on the shelf. Those values are typed in manually by me, not much difference to the original really.

Comment (variableundefined) [reply _maps/map_files/cyberiad/cyberiad.dmm]: OK

**Pull request merged**